src/test/java/com/google/javascript/gents/multiTests/import_rewrite/import.js

```diff
@@ -5,6 +5,10 @@ var D = goog.require("both.D");
 goog.require("nested.E");
 goog.require("nested.E.F");
 var stuff = goog.require("nested.E.F.Z");
+/** @suppress {extraRequire} My comment */
+goog.require("unused.W");
+/** @suppress  {extraRequire}  */
```

Comment:
Odd spaces here, is that intentional?

Reply:
Yes, making sure that space correctly gets matched by regex.

Reply:
Add a comment for the comment :) Otherwise, someone might helpfully fix this later.

Reply:
Ok, I took a look at google3 and there are 0 instances of multiple whitespaces there, so I think I'll just make this normal whitespace.

Reply:
sounds good. I am merging this PR.